tests/backends/tests.py

```diff
@@ -55,6 +55,11 @@ def test_initialization_class_attributes(self):
             instance_attr_value = getattr(conn, instance_attr_name)
             self.assertIsInstance(instance_attr_value, class_attr_value)
 
+    @unittest.skipUnless(connection.vendor != 'unknown', 'Test only if vendor is specified')
```

Comment:
The skip condition should be removed as it's not possible to run the tests with an unknown vendor, I think.